app/controllers/my_modules_controller.rb

```diff
@@ -21,9 +23,17 @@ class MyModulesController < ApplicationController
 
   def new
     @my_module = @experiment.my_modules.new
+
+    assigned_users = @experiment.project.manually_assigned_users
```

Comment:
maybe we can make something like this
```
assigned_users = User.where(id: @experiment.user_assignments.select(:user_id))
```
I think we need all users that have access to this experiment.